pkg/pool-hooks/test/foundry/ExitFeeHookExample.t.sol

```diff
@@ -58,7 +58,8 @@ contract ExitFeeHookExampleTest is BaseVaultTest {
         roleAccounts.poolCreator = lp;
 
         LiquidityManagement memory liquidityManagement;
-        liquidityManagement.disableUnbalancedLiquidity = true;
+        liquidityManagement.disableAddLiquidityUnbalanced = true;
```

Comment:
This one can actually be enabled right? That's the main use-case for this :)

Reply:
Yes, but I didn't want also change the example in this PR (and also update the tests, etc.)